internal/stacks/stack_manager.go

```diff
@@ -256,6 +263,11 @@ func (s *StackManager) ensureDirectories() error {
 			return err
 		}
 	}
+
+	if err := os.MkdirAll(filepath.Join(dataDir, "prometheus_config"), 0755); err != nil {
```

Comment:
Is this directory being used at all? It looks like the directory is created, but the config itself gets written to `configs`, which is actually the correct place for it. The `data` directory will get deleted and recreated if someone runs `ff reset` on the stack.

Reply:
Oh I see, so the `CopyFileToVolume` will run every time after a reset? If thats the case then yep this likely isn't needed. Will remove and test.

Reply:
Got it reading the code `reset` make its so that next time `start` is run, `runFirstTimeSetup` will be called 👍🏻

testing this now.